app/controllers/partners/users_controller.rb

```diff
@@ -40,6 +41,10 @@ def create
   private
 
   def find_account
-    @account = @partner.providers.find(params[:provider_id])
+    @account = @partner.providers.find(permitted_params[:provider_id])
+  end
+
+  def permitted_params
+    params.permit(%i[api_key email first_name last_name open_id username provider_id id page])
```

Comment:
I do not see why `api_key` is listed here

Reply:
when I checked params api_key is there so I listed here to not miss any key.

Reply:
What did you check? We do not use `permitted_params[:api_key]` or pass `permitted_params` in `User.new` or `user.update` so it should not be dangerous at all.

Reply:
I checked running specs like what all params will come
And permitted all the listed ones